package.json

```diff
@@ -23,7 +23,7 @@
     "develop": "export EDITOR_WARNINGS_OFF='true' && gatsby clean && gatsby develop",
     "start": "npm run develop",
     "edit": "unset EDITOR_WARNINGS_OFF && gatsby clean && gatsby develop",
-    "build": "gatsby build --prefix-paths",
+    "build": "npm run clean && gatsby build --prefix-paths",
```

Comment:
Yeah, I should have done this 4 months ago. While we're here, can you also add this to "start"?